packages/ckeditor5-indent/src/indentblockcommand.ts

```diff
@@ -50,13 +50,15 @@ export default class IndentBlockCommand extends Command {
 	 * @inheritDoc
 	 */
 	public override refresh(): void {
-		// Check whether any of the position's ancestors is a list item.
 		const editor = this.editor;
 		const model = editor.model;
 
 		const block = first( model.document.selection.getSelectedBlocks() );
 
-		if ( !block || !model.schema.checkAttribute( block, 'blockIndent' ) ) {
+		// Command should be disabled for block items in Document List items. See https://github.com/ckeditor/ckeditor5/issues/14155.
+		const isForwardAndInListItem = this._indentBehavior.isForward && block && block.hasAttribute( 'listItemId' );
```

Comment:
Lets use `DocumentListUtils.isListItemBlock()` instead.  Add a code comment about why only forward is disabled.

Reply:
Done.